Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN npm install -g pnpm@8.3.1
 
 COPY . .
 
+RUN apk add --update --no-cache python3 g++ make py3-pip
```

Comment:
why we need those package?

Reply:
no prebuilt package for gyp showed up for arm64, so it required python, make etc to make locally.

Reply:
I am not sure about it because in my another repo, which also build for arm64 but not add this package.

https://github.com/msgbyte/tailchat/blob/master/Dockerfile

Reply:
Okay, you are right. i start a test action and its looks need self-compiled..

maybe i using lts-alpine. and lts now is 20

Reply:
https://github.com/msgbyte/tianji/actions/runs/7490590412/job/20389816277